### PR TITLE
Update command-line-tools.rst

### PR DESCRIPTION
**Pull request by wiersgallak**
- Channel search: https://github.com/mattermost/mattermost-server/pull/10093
 - Team rename: https://github.com/mattermost/mattermost-server/pull/10083
 - Webhook show: https://github.com/mattermost/mattermost-server/pull/10123

### CODE REVIEW

File: source/administration/command-line-tools.rst [hunk 1]
```diff
@@ -358,6 +359,29 @@ mattermost channel restore
 
       ./mattermost channel restore 8soyabwthjnf9qibfztje5a36h
       ./mattermost channel restore myteam:mychannel
+      
+mattermost channel search
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Search for a channel by channel name.
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: What's the output? Is it to search if a channel already exists? Or to return the channel ID?

File: source/administration/command-line-tools.rst [hunk 2]
```diff
@@ -358,6 +359,29 @@ mattermost channel restore
...
+    
+  Format
+    .. code-block:: none
+
+      mattermost channel search {channelName}
+
+  Examples
+    .. code-block:: none
+
+      ./mattermost channel search mychannel
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: Are the accepted values for `mychannel` channel name only, or also channel id?

File: source/administration/command-line-tools.rst [hunk 3]
```diff
@@ -1814,6 +1861,22 @@ mattermost webhook modify-outgoing
 	  --url [callbackURL]           Callback URL 
 	  --content-type string         Content type 
 
+mattermost webhook show
+~~~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Show information about a webhook by providing the webhook ID. Returns display name, channel ID and Team ID for both incoming and outgoing webhooks.  Additionally returns callback URL, username, and icon URL for outgoing webhooks.
```
Comment (jasonblais) [inline source/administration/command-line-tools.rst]: `Team` > `team`

File: source/administration/command-line-tools.rst (hunk 2 above)
Comment (wiersgallak) [reply source/administration/command-line-tools.rst]: Channel id is not supported at this time.

File: source/administration/command-line-tools.rst (hunk 1 above)
Comment (wiersgallak) [reply source/administration/command-line-tools.rst]: updated the description to include the outputs.

**Comment by wiersgallak** [comment]
@jasonblais  - feedback addressed.  Thank you!

**Pull request merged**